engines/query-sparql-file/package.json

```diff
@@ -282,6 +282,7 @@
     "@comunica/actor-rdf-update-quads-hypermedia": "^3.2.1",
     "@comunica/actor-rdf-update-quads-rdfjs-store": "^3.2.1",
     "@comunica/actor-term-comparator-factory-expression-evaluator": "^3.2.1",
+    "@comunica/bus-function-factory": "^3.2.1",
```

Comment:
Are you sure these deps are required here? (they might, just checking to make sure)

Reply:
If they are not here, depcheck is not happy. I think it's necessary since the actors inherit from the dedicated actor which contains code?

Reply:
But then I'd expect that only the actors would need a dependency on this bus.

But if the depcheck says it needs to be there, it's probably right.

Reply:
No clue, but the same holds for bus query operation. I suspect it's fine.

Reply:
Ok!